### DISCUSSION TITLE
Great work

### DISCUSSION THREAD

**Discussion by PoC-dev** [fstark/macflim#2] [General]
Great stuff! I just tried that with my SE/30, with the flim located on an AppleTalk File Server connected via 10M Ethernet to my SE/30. If I lower the buffering to one frame, it's smoothly slow(er than from disk of course).

Btw., I'd strongly recommend that you might also publish an email address to contact you in an old-fashioned but privacy-enhanced way. All this shiny new social network stuff is not everybody's thing.

Thanks for Flim! I'll watch out for more to come.

**Comment by fstark** [comment]
Speaking of of shiny stuff that isn't for eveybody, I did not saw your message until today :-)

Thanks for you interest!

My real email is frederic.stark at gmail.com.
